src/pkg/cli/client/byoc/gcp/stream.go

```diff
@@ -91,12 +91,13 @@ func isContextCanceledError(err error) bool {
 	return false
 }
 
-func (s *ServerStream[T]) Start(start time.Time) {
+func (s *ServerStream[T]) Start(start time.Time, follow bool) {
```

Comment:
Best to avoid adding `bool` args to public/exported funcs. Easy fix is to create two public funcs and call a private one:
```suggestion
func (s *ServerStream[T]) StartFollow(start time.Time) {
	return s.start(start, true)
}

func (s *ServerStream[T]) Start(start time.Time, follow bool) {
	return s.start(start, false)
}

func (s *ServerStream[T]) start(start time.Time, follow bool) {
```